assets/js/components/setup/SetupUsingProxyWithSignIn/index.js

```diff
@@ -127,8 +134,13 @@ export default function SetupUsingProxyWithSignIn() {
 	return (
 		<Fragment>
 			<Header />
-			<div className="googlesitekit-setup">
-				<Grid>
+			<div
+				className={ classnames( 'googlesitekit-setup', {
+					'googlesitekit-setup-content-wrapper':
```

Comment:
Please use the BEM modifier syntax for this class name, it shouldn't be named as a new top level block.

`--wrapper` won't work so well as a modifier, maybe something like `--with-background` would be more appropriate.


```suggestion
					'googlesitekit-setup-content--with-background':
```